src/linearization.jl

```diff
@@ -38,6 +77,48 @@ function build_matrix(F₁, F₂, N)
     end
 end
 
+"""
+    lift_vector(x0, N)
+
+returns a vector of monomials in x0 (hyperrectangle) of degree at most N
+"""
+function lift_vector(x0, N)
+    monoms = generate_monomials(dim(x0), N)
+    nonzero_monoms = firstrest(monoms)[2]
+    result = []
+    intervals = [interval(low(x0, i), high(x0, i)) for i in 1:dim(x0)]
```

Comment:
I think here you can just do this with `LazySets.Interval`:
```suggestion
    intervals = [Interval(low(x0, i), high(x0, i)) for i in 1:dim(x0)]
```

Reply:
And then below when you do `i.lo` resp. `i.hi` you need to do `low(i, 1)` resp. `high(i, 1)`.

Reply:
@schillic Thank you for the suggestion!
I tried to do this but it seems that there is no `^` operator for `LazySets.Interval`. And using iterative product is not that precise because one gets
```julia
a = LazySets.Interval(-1, 1)
a * a # return (-1, 1), not (0, 1)
```
Am I missing something?
I can make a small PR to LazySets with this operator.

Reply:
Oh, I think it is okay to stick with `IntervalArithmetic` then.